man/systemd.xml

```diff
@@ -1487,7 +1485,26 @@
   </refsect1>
 
   <refsect1>
-    <title>Sockets and FIFOs</title>
+    <title>System clock epoch</title>
+
+    <para>When <command>systemd</command> is started or restarted, it may set the system clock to the
+    "epoch". This mechanism is used to ensure that the system clock remains somewhat reasonably initialized
+    and roughly monotonic across reboots, in case no battery-backed local RTC is available or it does not
```

Comment:
If the goal is to keep it monotonic *across reboots*, can first boots be excluded from this heuristics?

Scenario I have in mind: fresh installations, e.g., a software archeologist booting a CentOS 7 machine to compile a manylinux1 wheel about a decade from now.

Reply:
I don't think this would be a good idea. We certainly want to set the clock on first boot.

The archeologist will have to install an old version of systemd, reset the time explicitly after boot, or compile systemd with an old epoch set.